### ISSUE TITLE
Icon request: icon-play-backward

### ISSUE THREAD

**Issue by tomaspetterssonsmhi** [FortAwesome/Font-Awesome#8595]
Hi!

I would be great to have a new icon in the Video Player Icons category,  a play backward icon. I would look just as fa-play but arrow pointing left instead of right. 

I would use it in my web application for backward playing timesteps of WMS layers as an animation.

**Comment by tagliala** [comment]
Your request is legit, but also consider using `fa-flip-horizontal`

https://jsfiddle.net/tagliala/syduuue9/21/

**Comment by tomaspetterssonsmhi** [comment]
Yes its easy to just flip it. However, I think it would be used if available.
Thanks

**Comment by tagliala** [comment]
Reopening here

**Comment by aethior** [comment]
*Related Unicode Emoji* #222
◀ [U+25C0, ️U+FE0F](https://emojipedia.org/emoji/%E2%97%80/) Reverse Button
Also known as `Left Triangle`, `Left-Pointing Triangle` and `Black Left-Pointing Triangle`.

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂

**Comment by jabronus** [comment]
This icon is now part of Font Awesome 7: https://fontawesome.com/icons/play-flip?f=classic&s=solid
